nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/linalg/custom/CustomOpsTests.java

```diff
@@ -1353,4 +1351,72 @@ public void testCreateOp_1() {
 
         assertEquals(exp, result);
     }
+
+    @Test
+    public void testRgbToHsv() {
+        INDArray expected = Nd4j.createFromArray(new float[]{6.75000000e+01f, 2.54545455e-01f, 8.62745098e-01f, 1.80000000e+02f,
+                3.27777778e-01f, 7.05882353e-01f, 1.35066079e+02f, 9.26530612e-01f,
+                9.60784314e-01f, 7.45341615e-01f, 6.85106383e-01f, 9.21568627e-01f,
+                2.78688525e+02f, 7.85407725e-01f, 9.13725490e-01f, 2.10989011e+01f,
+                4.76439791e-01f, 7.49019608e-01f, 2.89038462e+02f, 8.48979592e-01f,
+                9.60784314e-01f, 1.56416185e+02f, 6.92000000e-01f, 9.80392157e-01f,
+                3.52881356e+02f, 5.31531532e-01f, 4.35294118e-01f, 1.07142857e+01f,
+                2.90155440e-01f, 7.56862745e-01f, 3.43384615e+02f, 3.86904762e-01f,
+                6.58823529e-01f, 1.78321678e+02f, 7.48691099e-01f, 7.49019608e-01f,
+                2.30645161e+02f, 7.78242678e-01f, 9.37254902e-01f, 3.19159664e+02f,
+                7.62820513e-01f, 6.11764706e-01f, 2.10126582e+01f, 9.71311475e-01f,
+                9.56862745e-01f, 2.90896552e+02f, 5.96707819e-01f, 9.52941176e-01f,
+                1.74822335e+02f, 9.42583732e-01f, 8.19607843e-01f, 2.06600985e+02f,
+                9.90243902e-01f, 8.03921569e-01f, 1.06883721e+02f, 8.70445344e-01f,
+                9.68627451e-01f, 1.95272727e+02f, 6.11111111e-01f, 7.05882353e-01f}).reshape(5,4,3);
+        INDArray input = Nd4j.createFromArray(new float[]{213.f, 220.f, 164.f, 121.f, 180.f, 180.f,  18.f, 245.f,  75.f, 235.f,  76.f,  74.f, 168.f,
+                50.f, 233.f, 191.f, 132.f, 100.f, 207.f,  37.f, 245.f,  77.f, 250.f, 182.f, 111.f,  52.f,
+                59.f, 193.f, 147.f, 137.f, 168.f, 103.f, 121.f,  48.f, 191.f, 187.f,  53.f,  82.f, 239.f,
+                156.f,  37.f, 118.f, 244.f,  90.f,   7.f, 221.f,  98.f, 243.f,  12.f, 209.f, 192.f,   2.f,
+                115.f, 205.f,  79.f, 247.f,  32.f,  70.f, 152.f, 180.f}).reshape(5,4,3);
+        RgbToHsv op = new RgbToHsv(input);
+        INDArray[] ret = Nd4j.exec(op);
+        assertEquals(ret[0], expected);
+    }
```

Comment:
For these sorts of tests, can you include info about how to reproduce them?
For example a few lines of python as a comment assuming you pulled it out of TF.
The problem with these sorts of tests is that if they fail, we have no idea if it's a bad test, some edge case, or an actual op failure.

Reply:
Added small comments.